docs/source/contributing/overview.md

```diff
@@ -1,150 +1,125 @@
+There are several ways you can contribute:
 
-## Linting
+- Identify and report any issues or bugs.
+- Suggest or implement new features.
+- Improve documentation or contribute a how-to guide.
 
-You can install the linting requirements using either `uv` or `pip`.
+## Issues
 
-Using `uv`:
-
-```bash
-uv sync --frozen --group lint --active --inexact
-```
-
-Using `pip`:
-
-```bash
-uv pip compile --group lint > requirements-lint.txt
-pip install -r requirements-lint.txt
-```
+If you encounter a bug or have a feature request, please search [existing issues](https://github.com/vllm-project/vllm-spyre/issues?q=is%3Aissue) first to see if it has already been reported. If not, please [create a new issue](https://github.com/vllm-project/vllm-spyre/issues/new/choose), providing as much relevant information as possible.
 
-After installing the requirements, run the formatting script:
-
-```bash
-bash format.sh
-```
-
-Then, make sure to commit any changes made by the formatter:
-
-```bash
-git add .
-git commit -s -m "Apply linting and formatting"
-```
-
-Now, you’re good to go! 🚀
-
-## DCO and Signed-off-by
-
-When contributing changes to this project, you must agree to the [DCO](https://github.com/vllm-project/vllm/blob/main/DCO).
-Commits must include a `Signed-off-by:` header which certifies agreement with
-the terms of the DCO.
-
-Using `-s` with `git commit` will automatically add this header.
+You can also reach out for support in the `#sig-spyre` channel in the [vLLM Slack](https://inviter.co/vllm-slack) workspace.
 
 ## Testing
 
-### Running tests locally on CPU (No Spyre card)
+### Testing Locally on CPU (No Spyre card)
   
-1. (arm64 only) Install xgrammar
+1. Install `xgrammar` (only for `arm64` systems):
   
    :::{tip}
-   It's installed for x86_64 automatically.
+   `xgrammar` is automatically installed on `x86_64` systems.
    :::
 
    ```sh
    uv pip install xgrammar==0.1.19
    ```
 
-1. (optional) Download `JackFram/llama-160m` model for tests
+1. (Optional) Download the `JackFram/llama-160m` model:
 
    ```sh
    python -c "from transformers import pipeline; pipeline('text-generation', model='JackFram/llama-160m')"
    ```
 
    :::{caution}
-   Downloading the same model using HF API does not work locally on `arm64`.
+   The Hugging Face API download does **not** work on `arm64`.
    :::
 
-   :::{tip}
-   :class: dropdown
-   We assume the model lands here:
+   By default, the model is saved to:
 
    ```sh
    .cache/huggingface/hub/models--JackFram--llama-160m
    ```
-  
-    :::
 
-1. Source env variables needed for tests
+1. Source environment variables:
 
    ```sh
    source _local_envs_for_test.sh
    ```
 
-1. (optional) Install dev dependencies (if vllm-spyre was installed without them)
+1. (Optional) Install development dependencies:
   
    ```sh
    uv pip install --group dev
    ```
 
-1. Run the tests:
+1. Run the tests using [markers](https://github.com/vllm-project/vllm-spyre/blob/main/pyproject.toml#L119):
```

Comment:
Can you do your docs magic and actually import the code from the file instead? The problem with using line numbers is that once the file changes then this link just points to something else entirely.